test/__mocks__/axios.js

```diff
@@ -0,0 +1,30 @@
+const axios = jest.genMockFromModule('axios');
```

Comment:
its ok to use jest for mocking, but i think we can refactor those ifs for something more extensible :)